Sources/StreamChatUI/Appearance+Formatters/MarkdownFormatter.swift

```diff
@@ -43,7 +44,76 @@ open class DefaultMarkdownFormatter: MarkdownFormatter {
 
     open func format(_ string: String) -> NSAttributedString {
         let markdownFormatter = SwiftyMarkdown(string: string)
-        markdownFormatter.code.fontName = Attributes.Code.fontName
+        modify(swiftyMarkdownFont: markdownFormatter.code, with: styles.codeFont)
+        modify(swiftyMarkdownFont: markdownFormatter.body, with: styles.bodyFont)
+        modify(swiftyMarkdownFont: markdownFormatter.link, with: styles.linkFont)
+        modify(swiftyMarkdownFont: markdownFormatter.h1, with: styles.h1Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h2, with: styles.h2Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h3, with: styles.h3Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h4, with: styles.h4Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h5, with: styles.h5Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h6, with: styles.h6Font)
         return markdownFormatter.attributedString()
     }
+
+    private func modify(swiftyMarkdownFont: FontProperties, with font: MarkdownFont) {
+        if let fontName = font.name {
+            swiftyMarkdownFont.fontName = fontName
+        }
+        if let fontSize = font.size {
+            swiftyMarkdownFont.fontSize = fontSize
+        }
+        if let fontStyle = font.styling?.asSwiftyMarkdownFontStyle() {
+            swiftyMarkdownFont.fontStyle = fontStyle
+        }
+    }
+}
+
+// TODO: Should we put this inside the DefaultFormatter?
+public struct MarkdownStyles {
```

Comment:
Lets show an example in the iMessage Demo App for example, and initialise the default formatter with different styling. 

Additionally, we probably can add `color` to the `MarkdownFont` since that is probably going to be requested a lot.

Then on iMessage Demo App we can do something like this:

```swift
let defaultFormatter = DefaultMarkdownFormatter()
defaultFormatter.styles.h1Font.color = .systemBlue
defaultFormatter.styles.h2Font.color = .systemBlue
defaultFormatter.styles.h3Font.color = .systemBlue
defaultFormatter.styles.h4Font.color = .systemBlue
defaultFormatter.styles.h5Font.color = .systemBlue
defaultFormatter.styles.h6Font.color = .systemBlue
```

Reply:
Updated 👍
Check out the latest commit